dynamixel_driver/src/dynamixel_driver/dynamixel_serial_proxy.py

```diff
@@ -136,6 +136,13 @@ def __fill_motor_parameters(self, motor_id, model_number):
         encoder_resolution = DXL_MODEL_TO_PARAMS[model_number]['encoder_resolution']
         range_degrees = DXL_MODEL_TO_PARAMS[model_number]['range_degrees']
         range_radians = math.radians(range_degrees)
+        if angles['max'] == 4095 and angles['min'] == 4095:  # In multi-turn mode if the servo is MX motor
```

Comment:
why not check `DXL_MODEL_TO_PARAMS` on `model_number` to see if multi-turn is supported rather than relying on this `try .. except`?